src/py21cmfast/src/SpinTemperatureBox.c

```diff
@@ -1746,8 +1747,8 @@ void ts_main(float redshift, float prev_redshift, float perturbed_field_redshift
             Ts_ave += ts_cell.Ts;
             Tk_ave += ts_cell.Tk;
             J_LW_ave += ts_cell.J_21_LW;
-            eps_lya_inj_ave += rad.dstarlya_cont_dt;
-            eps_lya_cont_ave += rad.dstarlya_inj_dt;
+            eps_lya_inj_ave += rad.dstarlya_inj_dt;
+            eps_lya_cont_ave += rad.dstarlya_cont_dt;
```

Comment:
Wow nice catch lol

Reply:
Thankfully it only affected the debug printing